FlashEnglish/Sources/Views/Home/HomeView.swift

```diff
@@ -25,7 +25,11 @@ struct HomeView: View {
         LevelItem(image: Asset.Assets.imgExpert.swiftUIImage, title: "専門家レベル", levelCase: .expert),
         LevelItem(image: Asset.Assets.imgMonster.swiftUIImage, title: "人外レベル", levelCase: .monster)
     ]
-    private let gridItems = [GridItem(.flexible()), GridItem(.flexible())]
+
+
+
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Masaya1582/FlashEnglish/blob/847505509e81e65488d4c86b3ee8c3c3d0172f5a/FlashEnglish/Sources/Views/Home/HomeView.swift#L31"></span>Limit vertical whitespace to a single empty line; currently 4<br />
<code>vertical_whitespace</code> <code>HomeView.swift:31</code></td>
    </tr>
  </tbody>
</table>